cmd/hyperkube/main.go

```diff
@@ -84,11 +84,11 @@ func commandFor(basename string, defaultCommand *cobra.Command, commands []func(
 }
 
 // NewHyperKubeCommand is the entry point for hyperkube
-func NewHyperKubeCommand(stopCh <-chan struct{}) (*cobra.Command, []func() *cobra.Command) {
+func NewHyperKubeCommand() (*cobra.Command, []func() *cobra.Command) {
 	// these have to be functions since the command is polymorphic. Cobra wants you to be top level
 	// command to get executed
 	apiserver := func() *cobra.Command {
-		ret := kubeapiserver.NewAPIServerCommand(stopCh)
+		ret := kubeapiserver.NewAPIServerCommand(server.SetupSignalHandler)
```

Comment:
Just call `kubeapiserver.NewAPIServerCommand(server.SetupSignalHandler)` here and don't modify the other files? You'll get your laziness without mucking with the signatures of the other app packages.

Reply:
Do you mean `kubeapiserver.NewAPIServerCommand(server.SetupSignalHandler())`? This can't be done as all command functions are called during `commandFor`.